uwblueprint/community-fridge-kw
# feat: add empty state to calendar

## Brief description. What is this change? 

### [Add “empty state” for calendar](https://www.notion.so/uwblueprintexecs/Add-empty-state-for-calendar-d6d698bfcdd744918d4ab29d32ba9759)



## Implementation description. How did you make this change?
* Added an “empty state” card for days without shifts (on public calendar)
     * Design confirmed with Anya and Stacy
* Update image for reset password email verification page

![image](https://user-images.githubusercontent.com/69697180/164170293-bd8c7be4-1fb0-4e4c-a06e-6da36c94a00c.png)


## Steps to test
1. Sign in as donor
2. Schedule shifts and check that the public calendar accurately displays shifts
     * None on Apr 23 - empty state
     * One shift on Apr 23 - one card
     * Another shift on Apr 23 - two cards




## Checklist
